perfkitbenchmarker/benchmark_spec.py

```diff
@@ -139,59 +195,35 @@ def ConstructVirtualMachines(self):
     """Constructs the BenchmarkSpec's VirtualMachine objects."""
     vm_group_specs = self.config.vm_groups
 
-    zone_index = 0
+    os_type = vm_group_specs[vm_group_specs.keys()[0]].os_type
+    if os_type == os_types.JUJU:
```

Comment:
Users can assign arbitrary os_types to each group via config files - I haven't seen anyone using it that way, but the capability exists so I'd like to continue supporting it as much as possible